neuraxle/checkpoints/pickle_checkpoint_step.py

```diff
@@ -0,0 +1,73 @@
+"""
+Neuraxle Pickle Checkpoint Step
+====================================
+Checkpoint step that loads and saves pickles as checkpoints
+..
+    Copyright 2019, Neuraxio Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import os
+import pickle
+
+from neuraxle.checkpoints.base_checkpoint_step import BaseCheckpointStep
+
+DEFAULT_CACHE_FOLDER = os.path.join(os.getcwd(), '/cache/')
```

Comment:
The os.path.join will refuse the slashes "/" in the `os.path.join(os.getcwd(), '/cache/')`. Replace with `os.path.join(os.getcwd(), 'cache')`.